ros/riberry_startup/node_scripts/teaching_mode.py

```diff
@@ -68,113 +61,88 @@ def button_cb(self, msg):
-                av = np.array(list(motion['joint_states'].values()))
-                avs.append(av)
-                tms.append(current_time - prev_time)
-            prev_time = current_time
-        rospy.loginfo('angle vectors')
-        rospy.loginfo(f'{avs}')
-        rospy.loginfo('times')
-        rospy.loginfo(f'{tms}')
-        self.ri.angle_vector_sequence(avs, tms)
-        # Check interruption by button
-        # use self.ri.is_interpolating() after the following PR is merged
-        # https://github.com/iory/scikit-robot/pull/396
-        def is_interpolating(controller_type=None):
-            if controller_type:
-                controller_actions = self.ri.controller_table[controller_type]
+            if self.play_file is None:
+                # select play file
+                if msg.data == 1:
+                    self.play_list.increment_index()
+                # confirm play file
+                elif msg.data == 2:
+                    self.play_file = self.play_list.selected_option()
+                    if self.play_file is None:
+                        self.state = State.WAIT
+                # delete play file
+                elif msg.data == 3:
+                    delete_file = self.play_list.selected_option()
+                    if delete_file is None:
+                        self.state = State.WAIT
+                    if os.path.exists(delete_file):
+                        os.remove(delete_file)
+                    self.play_list.remove_option(delete_file)
+                    self.state = State.WAIT
             else:
-                controller_actions = self.ri.controller_table[self.ri.controller_type]
-            is_interpolatings = (action.is_interpolating() for action in controller_actions)
-            return any(list(is_interpolatings))
-        while not rospy.is_shutdown() and is_interpolating():
-            if self.state == State.WAIT:
-                self.ri.cancel_angle_vector()
-                rospy.loginfo('Play interrupted')
-                break
-            rospy.sleep(0.5)  # Save motion every 0.5s to smooth motion play
-        rospy.loginfo('Play finished')
+                # stop playing
+                if msg.data == 2:
+                    self.motion_manager.stop()
+                    self.state = State.WAIT
 
     def timer_callback(self, event):
         if self.mode != "TeachingMode":
             return
         sent_str = ''
         if self.state == State.WAIT:
             sent_str += 'Teaching mode\n\n'\
-                + 'single click:\n  record\n\n'\
-                + 'double click:\n  play\n\n'\
-                + 'triple click:\n  servo_off'
+                + '1tap:\n record\n\n'\
+                + '2tap:\n play\n\n'\
+                + '3tap:\n servo_off'
         elif self.state == State.RECORD:
             sent_str += 'Record mode\n\n'\
-                + 'single click:\n  stop recording'
+                + '1tap:\n  finish recording'
```

Comment:
`finish recording` is a bit long.
I think `finish` is enough. (like other sent_str)